### PR TITLE
Itk 5.4rc03+patches+pyproject.toml-stable-abi-wheels--macos-arm

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-ants"
```
Comment (dzenanz) [inline pyproject.toml:6]:
```suggestion
name = "itk-elastix"
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    { name = "Tom Birdsong" },
+    { name = "Konstantinos Ntatsis", email = "konstantinos.ntatsis323@gmail.com" },
+]
+keywords = [
+    "itk",
+    "elastix",
+    "registration",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
```
Comment (dzenanz) [inline pyproject.toml:24]: Should we bump this up to some more mature level?

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+[project.urls]
+Download = "https://github.com/InsightSoftwareConsortium/ITKElastix"
+Homepage = "https://github.com/InsightSoftwareConsortium/ITKElastix"
+
+[tool.scikit-build]
+# The versions of CMake to allow. If CMake is not present on the system or does
+# not pass this specifier, it will be downloaded via PyPI if possible. An empty
+# string will disable this check.
+cmake.version = ">=3.16"
```
Comment (dzenanz) [inline pyproject.toml:55]:
```suggestion
cmake.version = ">=3.16.3"
```

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+[project]
+name = "itk-ants"
+version = "0.19.2"
+description = "Provides an ITK Python interface to elastix, a toolbox for rigid and nonrigid registration of images"
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Matthew M. McCormick", email = "matt.mccormick@kitware.com" },
+    { name = "ViktorvdValk", email = "viktorvandervalk@gmail.com" },
```
Comment (N-Dekker) [inline pyproject.toml:13]: @ViktorvdValk Please check!

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    { name = "Niels Dekker", email = "N.Dekker@lumc.nl" },
+    { name = "Tom Birdsong" },
+    { name = "Konstantinos Ntatsis", email = "konstantinos.ntatsis323@gmail.com" },
```
Comment (N-Dekker) [inline pyproject.toml:16]: @ntatsisk Please check!

File: pyproject.toml (hunk 4 above)
Comment (N-Dekker) [reply pyproject.toml:13]: Checked by Viktor, in person 😃

File: pyproject.toml (hunk 2 above)
Comment (thewtex) [reply pyproject.toml:24]: No, I think we are still beta.

**Review by dzenanz — APPROVED** [review]

**Pull request merged**